codespell_lib/data/dictionary.txt

```diff
@@ -297,6 +297,9 @@ accessiblilty->accessibility
 accessile->accessible
 accessintg->accessing
 accessisble->accessible
+accessoire->accessory
+accessoires->accessories, accessorise,
+accessoirez->accessorize, accessories,
 accessort->accessor
 accesss->access
 accesssor->accessor
```

Comment:
Could also suggest `accesssor->access or` (if that's not too far outside the scope of this PR)
Should  `accessor->access or` maybe be in the code dictionary?

Reply:
> Could also suggest `accesssor->access or` (if that's not too far outside the scope of this PR)

I think that should be a separate PR if it goes in, but you're saying you typed an s instead of a space seems rather unlikely to be. Some real world examples would be good again.

> Should `accessor->access or` maybe be in the code dictionary?

Yes if it was to go in. There seem to be enough examples of those words together that it's plausible. But probably one for a separate PR given this one is getting big. Why don't you do the honours?

Reply:
Sorry if I made too many suggestions, I'm happy for you to just ignore any comments I made, as you see fit.